src/app/login-register/service/user-register.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {FormControl} from "@angular/forms";
+import {User} from "../../classes/user";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserRegisterService {
+
+  public email: FormControl;
+  public password: FormControl;
+
+  constructor(private http: HttpClient) { }
+
+  registerUser(email: FormControl, password: FormControl){
+    const headers = new HttpHeaders()
+      .set(
+        'Content-Type',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
        "Content-Type",
```